command/pki_roletest.go

```diff
@@ -0,0 +1,152 @@
+package command
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/mitchellh/cli"
+	"github.com/posener/complete"
+)
+
+var (
+	_ cli.Command             = (*PKIRoleTestCommand)(nil)
+	_ cli.CommandAutocomplete = (*PKIRoleTestCommand)(nil)
+)
+
+type PKIRoleTestCommand struct {
+	*BaseCommand
+
+	flagQuiet bool
+	flagMount string
+}
+
+func (c *PKIRoleTestCommand) Synopsis() string {
+	return "Test PKI Secrets Engine role issuance restrictions"
+}
+
+func (c *PKIRoleTestCommand) Help() string {
+	helpText := `
+Usage: vault pki role-test [options] ROLE COMMON_NAME [DNS-SANS...] [K=V]
+
+  Reports whether or not issuance will succeed against the specified
```

Comment:
Remove one specified: ".. against the PKI role with the specified.."

Reply:
d'oh, thanks :-)